src/mngt/ocf_mngt_cache.c

```diff
@@ -2405,6 +2405,14 @@ static void _ocf_mngt_activate_compare_superblock_end(
 		OCF_PL_FINISH_RET(context->pipeline, result);
 
 	if (diff) {
+
+		result = env_memcmp(&cache->conf_meta->line_size, sizeof(cache->conf_meta->line_size),
```

Comment:
line_size is a simple integer, so it's easier to compare it with "==" rather than env_memcmp

Reply:
Yes, it seems to be much simpler.